src/main/java/com/onevizion/scmdb/DbManager.java

```diff
@@ -258,4 +254,46 @@ public void generateDdlForAllObjects() {
         ddlGenerator.executeSettingTransformParams();
         ddlGenerator.generateDllsForAllDbObjects();
     }
+
+    /**
+     * The method filters scripts by type, separates them into two parts (arr1 and arr2).
+     * The arr2 scripts are in the range 1..100, arr1 are all the others. If the type is ROOLBACK, then both parts are
+     * filtered in reverse order.
+     *
+     * Example 1 : Input  scripts = [1_script1.sql, 1_script1_rollback.sql, 8861_script2.sql, 8861_script2_rollback.sql]
+     *                    scriptType = COMMIT
+     *             Output List<SqlScript> = [8861_script2.sql, 1_script1.sql]
+     *
+     * Example 2 : Input  scripts = [1_script1.sql, 1_script1_rollback.sql, 8861_script2.sql, 8861_script2_rollback.sql]
+     *                    scriptType = ROLLBACK
+     *             Output List<SqlScript> = [1_script1_rollback.sql, 8861_script2_rollback.sql]
+     * @param scripts List of new scripts
```

Comment:
`list of scripts to sort`, no matter if it new or old

Reply:
Fixed.